tests/modules/eventloop.c

```diff
@@ -0,0 +1,157 @@
+#define REDISMODULE_EXPERIMENTAL_API
+#include "redismodule.h"
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <memory.h>
+
+int fds[2];
+long long buf_size;
+char *src;
+long long src_offset;
+char *dst;
+long long dst_offset;
+long long slow; /* If not zero, do a single read/write and return */
+
+RedisModuleBlockedClient *bc;
+RedisModuleCtx *reply_ctx;
```

Comment:
i didn't understand what this test does (didn't spend too much time on it).
maybe add some comment to describe it.

Reply:
Added more comments and tests